app/code/community/Doofinder/Feed/Model/Map/Product/Abstract.php

```diff
@@ -250,8 +250,7 @@ protected function getProductPrice($field)
         // Convert price to store currency
         $price = Mage::helper('core')->currencyByStore($price, $store, false, false);
 
-        // Format price with store currency
-        return $store->getCurrentCurrency()->format($price, array('display' => Zend_Currency::NO_SYMBOL), false);
+        return round($price, 2);
```

Comment:
This should return the price rounded to the current currency decimal positions or with no rounding at all if not possible.